eng/tools/RepoTasks/RepoTasks.csproj

```diff
@@ -15,14 +15,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NuGet.Packaging" Version="5.6.0" />
+    <PackageReference Include="NuGet.Packaging" Version="5.8.0" />
```

Comment:
@javiercn while you're addressing Doug's feedback please also upgrade this to 5.9.0

Reply:
@javiercn and I chatted offline. He's going to do a follow-up for the API baseline files. This can go there too.

Reply:
In that case I'll update it in https://github.com/dotnet/aspnetcore/pull/31064